XamlControlsGallery/ControlPages/AppBarButtonPage.xaml.cs

```diff
@@ -64,9 +63,7 @@ private void CompactButton_Click(object sender, RoutedEventArgs e)
 
         private void AppBarButton_Click(object sender, RoutedEventArgs e)
         {
-            Button b = sender as Button;
-
-            if (b != null)
+            if (sender is Button b)
```

Comment:
Nit: We might want to rename in this PR since `b` is not the best name for a variable and we are already touching this code.

Reply:
I'm definitely up for using better names. 

However, I feel changing single letter names to be more descriptive might be best left to its own commit/PR - rather than intermingling those changes with unrelated commits/PRs just because the code was touched.

Reply:
> However, I feel changing single letter names to be more descriptive might be best left to its own commit/PR - rather than intermingling those changes with unrelated commits/PRs just because the code was touched.

Agreed.

My philosophy on this is the same as my philosophy in life: _Change only one thing at a time_

Having a more descriptive name is a righteous thing to do, but I agree with @kmgallahan on this topic: we should save that for its own PR.

Reply:
Okay, yeah that's probably better.